src/utils.ts

```diff
@@ -87,10 +87,21 @@ const isRemoteConfigAvailable = (key: string) =>
 
 export const isABTestingAvailable = (key: string): Promise<boolean> => {
     if (!remoteConfig) {
-        return postMessageToNativeApp({type: 'GET_REMOTE_CONFIG'}).then(res => {
+        // If GET_REMOTE_CONFIG takes more than 5s to respond, resolve to false
+        const timeoutP = new Promise<boolean>(resolve => {
+            setTimeout(() => {
+                resolve(false);
+            }, 5000);
```

Comment:
To me this is too high, 500ms or less should be enough

Reply:
500ms can be too optimistic, in case this needs a network request, maybe 1 or 2 seconds is fine

Reply:
But this never waits for a network request it always returns the cached value or empty object

Reply:
ok, set to 500